_test/check-configlet-fmt.sh

```diff
@@ -9,6 +9,15 @@ configlet="${repo}/bin/configlet"
 c="${repo}/config.json"
 m="${repo}/config/maintainers.json"
 
+# Check if config.json was modified
+git diff-tree --no-commit-id --name-only -r HEAD^ | grep "$c"
```

Comment:
since this only checks `config.json` but `configlet fmt` modifies `maintainers.json` and `config.json`, I think this now makes it possible for a PR to give `maintainers.json` nonstandard formatting if it only modifies `maintainers.json` and not `config.json`. is that true?

Reply:
> Should I now add a check for maintainers.json?

Yes, I think it has to be. Since the goal is that the two files should have standard formatting